README.md

```diff
@@ -53,6 +53,20 @@ bundle exec jekyll build
 bundle exec htmlproofer _site
 ```
 
+### Code style
+
+[JavaScript Standard Style](https://standardjs.com/) is the authority on style for JavaScript files. This is enforced on pull requests to the main branch. Linting is run across all files as part of the test script.
```

Comment:
`JavaScript Standard Style](https://standardjs.com/) is the authority on style for JavaScript files` - is this true? I don't know what's more popular out of Standard and ESLint. I think Standard is our standard but I'm no longer sure how standard a standard it is 😅

It's listed as an example of a linter that enforces a standard style [here](https://github.com/dxw/tech-team-rfcs/blob/main/rfc-035-use-standard-linting-tools-across-all-our-projects.md), but it being an _example_ suggests that it's not the only option - the RFC is more about using some kind of standard rather than the Standard standard specifically

Reply:
Sorry I've used funny wording there - I meant on this project. I'll reword that so it sounds less like **the** authority!